test/denormalizer/ReduxApiStateDenormalizer.spec.js

```diff
@@ -321,6 +321,20 @@ describe('ReduxApiStateDenormalizer', () => {
       assert.isObject(cachedDenormalizedData[STATUS]);
       assert.isObject(cachedDenormalizedData['type2.test'][STATUS]);
     });
+    it('gets object from cache for unexisting item ', () => {
```

Comment:
Ok I get it, but it would be great if you could add additional test here showing that cache is working for identifiers over denormalizeOne